dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -77,8 +75,18 @@ internal class RumSessionScope(
 
             onApplicationDisplayed(event, viewScope, actualWriter)
             activeChildrenScopes.add(viewScope)
-        } else if (activeChildrenCount == 0) {
-            devLogger.w(MESSAGE_MISSING_VIEW)
+        } else if (activeChildrenScopes.size == 0 && (
+                event is RumRawEvent.AddError ||
```

Comment:
maybe it makes sense to split this condition in a way:

```
else if (size == 0) {
  if (event is ... || event is ... || ...) {
     ...
  } else {
     ...
  }
}
```

so that in the `else` we could add the warning which may be handy if another action is added in the codebase, which can start the scope, but we forgot to update this piece of code. wdyt?